src/snowflake/cli/api/project/schemas/project_definition.py

```diff
@@ -102,23 +104,53 @@ def _convert_env(
         return ProjectEnvironment(default_env=(env or {}), override_env={})
 
 
-class ProjectDefinition(_DefinitionV11):
-    def __init__(self, **kwargs):
-        super().__init__(**kwargs)
-        self._validate(kwargs)
+class DefinitionV20(_ProjectDefinitionBase):
+    entities: Dict = Field(
```

Comment:
Is there any option to make this `Dict[str, Entity]`, `Entity` being a pydantic base for future enitites?

Reply:
This is just a basic schema at this point. Let's discuss the actual pydantic schemas in the [next PR](https://github.com/snowflakedb/snowflake-cli/pull/1280).